docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -1867,7 +1867,13 @@ When `LOGGING` is enabled, attestation and block performance is reported as log
     ws-checkpoint: "0x5a642bb8f367e98c0d11426d98d28c465f8988fc960500886cb49faf0372883a:3600"
     ```
 
-A recent checkpoint within the [weak subjectivity period].
+A recent checkpoint within the [weak subjectivity period]. Accepts the checkpoint using either
+`<blockRoot>:<epochNumber>`, or a URL containing the `<blockRoot>:<epochNumber>` in a JSON payload.
```

Comment:
Might be helpful to describe the expected payload structure?  Or just paste an example of the json. 
```suggestion
`<blockRoot>:<epochNumber>`, or a URL which returns the `<blockRoot>:<epochNumber>` in a JSON payload via the key `ws_checkpoint`.
```

Also may be worth highlighting that the literal value (root:epoch) needs to start with "0x"